ls1intum/Artemis
# Detect Text Submission Language and display it to tutors

### Checklist


### Motivation and Context
I want to eliminate the problem that non german teaching asisstants have to assess german submissions
### Description
language was added to the submission model. ArTEMiS can now detect if a text submission is written in English or German

### Steps for Testing
1. Log in to ArTEMiS
2. Submit either a German or an English sentence as student solution to a text exercise.
3. Go to the Tutor Exercise Dashboard and Check if the correct Lanugage is written in the Language Collum


### Screenshots
![artemisscreen](https://user-images.githubusercontent.com/36135323/60769521-2623cc00-a0d1-11e9-874f-a32cb6fbce0d.png)
